qiita_db/study.py

```diff
@@ -263,7 +276,7 @@ def get_info(cls, study_ids=None, info_cols=None):
             if study_ids is not None and len(res) != len(study_ids):
                 raise qdb.exceptions.QiitaDBError(
                     'Non-portal-accessible studies asked for!')
-            return res
+            return [dict(r) for r in res]
```

Comment:
Sorry, that comment meant to go here.

I think the docstring should be updated here.